registry/publications.md

```diff
@@ -3,17 +3,15 @@ layout: doc
 title: Publications Related to the OBO Foundry
 ---
 
-## Papers about the OBO Foundry
+[The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration](http://www.nature.com/nbt/journal/v25/n11/abs/nbt1346.html)
```

Comment:
Hi Lynn, I'm confused because it looks like you took out the new 2021 paper, but I might be reading the diff wrong?

Reply:
@nlharris This update pulls in the last 50 commits, including #1634, not just DISDRIV.

Reply:
@nlharris Ahh, you're right, there is a problem. Your #1634 modified `registry/publications.md`, but that's an automatically generated file, and my update overwrote your changes. I'll fix this. Thanks!

Reply:
@beckyjackson fixed this, so I'll mark as 'resolved'.